source/dotnet/Library/AdaptiveCards/AdaptiveChoiceSetInput.cs

```diff
@@ -40,6 +40,7 @@ public class AdaptiveChoiceSetInput : AdaptiveInput
 #if !NETSTANDARD1_3
         [XmlAttribute]
 #endif
+        [DefaultValue(typeof(AdaptiveChoiceInputStyle), "compact")]
```

Comment:
Do we have no unit tests covering our default options?

Reply:
They currently cover roundtripping, which means that this type of failure isn't caught. I've created a `Known Gaps` section in the OneNote to document this.